tests/phpunit/includes/test-wc-amazon-payments-advanced-api.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Test cases for WC_Amazon_Payments_Advanced_API.
+ *
+ * @package WC_Gateway_Amazon_Pay/Tests
+ */
+
+declare(strict_types=1);
+
+/**
+ * WC_Amazon_Payments_Advanced_API_Test tests functionalities in WC_Amazon_Payments_Advanced_API.
+ */
+class WC_Amazon_Payments_Advanced_API_Test extends WP_UnitTestCase {
+
+	/**
+	 * Test gateway's default settings.
+	 *
+	 * @return void
+	 */
+	public function test_get_default_settings() : void {
```

Comment:
Why do we want to test the default settings? It's primarily static values, except `WC_Amazon_Payments_Advanced_API::get_payment_region_from_country( WC()->countries->get_base_country() )`. It that the part of business logic we want to test?

Reply:
Seemed like a good first test. I though it would also ensure settings being added to the plugin to have the correct default value. It would make us have a double look, since the test would fail otherwise.